spec/compiler/crystal/tools/init_spec.cr

```diff
@@ -113,14 +112,13 @@ dependencies:
 
         readme.should contain(%{TODO: Write a description here})
 
-        readme.should_not contain(%{1. Run `shards init`})
-        readme.should_not contain(%{2. Add the dependency to your `shard.yml`:})
+        readme.should_not contain(%{1. Add the dependency to your `shard.yml`:})
         readme.should_not contain(%{```yaml
 dependencies:
   example:
     github: jsmith/example
 ```})
-        readme.should_not contain(%{3. Run `shards install`})
+        readme.should_not contain(%{2. Run `shards`})
```

Comment:
Please keep the explicit `shards install`. The default action can change, and be removed to become a help message instead.

Reply:
Also it shows explicitly what this command is doing.